### PR TITLE
Milestone: Added logic

### PR DESCRIPTION
**Pull request by Roseokpe**
In this milestone, I added logic and tested it on the browser to be sure that it was working.

### CODE REVIEW

**Review by roshan-bajgain — APPROVED** [review]
Hi @Roseokpe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,39 +1,60 @@
...
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: 0,
+      next: '',
+      operation: '',
+    };
+    this.handleClicks = this.handleClicks.bind(this);
```
Comment (MeqdamAlqudah) [inline src/components/calculator.js:12]: - [ ] Please The Calculator component (components/Calculator.js) should be refactored from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html)
image
- should look like this
```javascript
// ./src/components/Calculator.js

import React from 'react';

const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);

export default Calculator; // export Calculator component
```

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Roseokpe , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by roshan-bajgain** [comment]
@MeqdamAlqudah, hello bro I review the event project and you review the hook on the same pull request. I am in confusing that she submitted the same pull request or ? I notice that you re-review the project who is the first reviewer?

File: src/components/calculator.js (hunk 1 above)
Comment (Roseokpe) [reply src/components/calculator.js:12]: @roshan-bajgain it was a mistake, I only wanted to request a review  for logic/handle event.

Comment (roshan-bajgain) [reply src/components/calculator.js:12]: @Roseokpe, I highly recommended you please create a new pull request for every review.

**Pull request merged**